src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Windows/Native/SafeHandles.cs

```diff
@@ -12,22 +13,19 @@ namespace Internal.Cryptography.Pal.Native
     /// </summary>
     internal abstract class SafePointerHandle<T> : SafeHandle where T : SafeHandle, new()
     {
-        public SafePointerHandle()
+        protected SafePointerHandle()
             : base(IntPtr.Zero, true)
         {
         }
 
-        public sealed override bool IsInvalid
+        internal static T CreateInvalid()
         {
-            get
-            {
-                return handle == IntPtr.Zero;
-            }
+            return new T();
         }
 
-        public static T InvalidHandle
+        public sealed override bool IsInvalid
         {
-            get { return new T(); }
+            get { return handle == IntPtr.Zero; }
```

Comment:
:bulb: This property didn't change. But the change to the brace layout means we all get to review it as though it did.